doc/STARTING_GUIDE.md

```diff
@@ -16,7 +16,26 @@ There are three steps necessary to start development, in this case for Android:
 
 1. `make run-clojure` - Compiles Clojure into JavaScript, watches for changes on cljs files, and hot-reloads code in the app
 2. `make run-metro` - Starts metro bundler and watches JavaScript code
-3. `make run-android` - Builds the Android app and starts it on the device
+3. `make run-android` - Builds the Android app and starts it on the device.
+
+> If you're on Mac and wish to develop for iOS you can use:
+
+- `make run-ios`
+
+> You can specify the simulator type by adding the SIMULATOR flag like this
+
+- `make run-ios SIMULATOR="iPhone 11 Pro"`
+
+> If you wish to work on a physical iPhone:
```

Comment:
This whole section does not belong in `Development, as it's supposed to be a quick rundown of main development targets.

Please add a subsection called `Physical iOS Device` or something like that in `Manual Steps` and move the steps related top UDID there.